### ISSUE TITLE
Error message regarding 'x' values

### ISSUE THREAD

**Issue by uklejaj** [aryeelab/hichipper#46]
Processing: DB-1-data
Error in { : task 1 failed - "need at least four unique 'x' values"
Calls: %do% -> <Anonymous>

Would you happen to know what is causing this issue?

**Comment by caleblareau** [comment]
This isn't a particularly informative error message, but typically this is caused by a sample where there is very very low sequencing depth. can you send a cat of your `*.stat` files?

**Comment by uklejaj** [comment]
[DB-1-data.txt](https://github.com/aryeelab/hichipper/files/2145896/DB-1-data.txt)
Here it is as a txt

**Comment by caleblareau** [comment]
Yeah you have way way too valid intrachromosomal interactions. I would say that this dataset is p

```
Mapped_unique_intra_quality_anchor=774
Mapped_unique_intra_quality_anchor_small=495
Mapped_unique_intra_quality_anchor_med=207
Mapped_unique_intra_quality_anchor_large=72
```

judging by how aggressively these statistics fall of, this looks like a failed library. hichipper can have issues when the data is this sparse
